### DISCUSSION TITLE
upgraded webpack 4 from 5.58.1 and getting error

### DISCUSSION THREAD

**Discussion by Sushmasrig** [webpack/webpack#14542] [Q&A]
# Bug report

**What is the current behavior?**

I upgraded webpack 4 to 5.58.1 and I am getting error: 
`config.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)
383.9f47b6c6525b4d9808d8.js?7e90ac95af08a307e9b7:2 TypeError: (0 , o.X) is not a function
TypeError: (0 , o.X) is not a function
   at 913.76620fac6614d6b7db5d.js:2
   at Object.useMemo (383.9f47b6c6525b4d9808d8.js:2)
   at e.useMemo (913.76620fac6614d6b7db5d.js:2)
   at s (913.76620fac6614d6b7db5d.js:2)
   at no (383.9f47b6c6525b4d9808d8.js:2)`

I tried general possible cases like Removing node_modules/package-lock.json reinstalling packages etc and I go through the releated github issues and few of similar error in stackoverflow as well but not getting success

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Other relevant information:**
webpack version:  5.58.1
Node.js version: 14.18.1
Operating System: windows 10
Additional tools:

**Comment by alexander-akait** [comment]
Can you provide configuration? Looks like some plugin break your code

**Comment by Sushmasrig** [comment]
removing this from webpack solves the issue.
   optimization: {
        splitChunks: {
            chunks: 'all',
        },
    },
